portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,124 @@
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+  width:1000px;
+  color:#777;
+  font-family:Verdana, Geneva, Tahoma, sans-serif;
+  background-color: lightblue;
+  margin-left:auto;
+  margin-right:auto;
 }
 
-#greeting-container {
+#pix {
+    text-align:center;
+    background-color: none;
+}
+
+h1{
+    text-align: center;
+    color:#777;
+}
+
+h2{
+    text-align: left;
+    font-weight: 500;
+    color:#777;
+}
+
+#p {
```

Comment:
Did you mean to style the `<p>` tag here? If so, let's change this to `p {`. If not, consider changing the name of the ID here and in the HTML usage to something else (e.g. `id=text` and `#text {`). Helps with readability of the code!